includes/specials/SpecialSearch.php

```diff
@@ -232,36 +240,6 @@ public function showResults( $term ) {
 
 		$out = $this->getOutput();
 
-		// Wikia change /Begin (ADi)
-		if ( ( $search instanceof SearchErrorReporting ) && $search->getError() ) {
```

Comment:
As far as I can tell `SearchErrorReporting` does not exist and `$search = $this->getSearchEngine()` cannot ever return it.